content/news/2023-03-04-bevy-0.10/index.md

```diff
@@ -587,6 +587,152 @@ query.iter().for_each(|mut component| {
 
 These abstractions were introduced in [#6404](https://github.com/bevyengine/bevy/pull/6404), [#7381](https://github.com/bevyengine/bevy/pull/7381) and [#7568](https://github.com/bevyengine/bevy/pull/7568).
 
+## `StandardMaterial` Blend Modes
```

Comment:
Sadly "code" formatting in headers is really bad in our CSS right now. Can you remove it for now?